app/views/assets/show.html.slim

```diff
@@ -6,24 +6,24 @@
   .form
 		p
   		strong Item model:
```

Comment:
You need to indent the things under the p by one indentation so they fall within the p paragraph